docs/protocol/agents/relayer.mdx

```diff
@@ -1,35 +1,83 @@
 # Relayer
 
-A relayer is responsible for ensuring messages are delivered to their recipients. Relayers are a permissionless but integral part of the Hyperlane protocol. Anyone can run a relayer!
+## Overview
+
+### Core Function
+The relayer runs the transport layer of the Hyperlane protocol by delivering messages from origin to destination chains. Depending on the [Interchain Security Module (ISM)](../../reference/ISM/specify-your-ISM) specified by the recipient, metadata may be required to prove the security of the message, such as Validator signatures for the [Multisig ISM](../ISM/multisig-ISM.mdx), merkle proofs, zero-knowledge proofs, and more. So before attempting to deliver, the relayer gathers any ISM-specific metadata.
+
+:::tip
+The relayer will periodically retry metadata gathering and message submission. This is important to improve resilience against disruptions, such as: no gas payment made to pay for delivery, insufficient relayer balance, temporary spikes in transaction fees on the destination, Validator downtime.
+:::
+
+### Relayer Incentives
+Message senders are charged for delivery on the origin chain, and it is the relayer operator's responsibility to rebalance revenue to destination chain accounts so delivery transaction fees can always be paid for. The relayer does not receive direct token incentives from the protocol, but operators can configure their fee structure for providing their critical service. For more information, check out [Interchain Gas Payments (IGPs)](../interchain-gas-payment.mdx).
+
+### Operating a Relayer
+Relayers can easily configure the messages that they wish to process. Currently, the relayer will support:
+
+1. A sender / recipient whitelist.
+2. A sender / recipient blacklist.
+3. The ability to accept [payments on the origin chain](../../reference/messaging/send.mdx#quote-dispatch) for processing a message on the destination chain.
+
+While permissionless to run, it's up to the message sender to pick which relayer to pay for delivery. [Mailbox](../mailbox.mdx) integrations usually default to always paying the same relayer, so if driven by economic incentives, it's only recommended to run a relayer for a self-deployed bridge. For convenience, Hyperlane will run an open source and configurable relayer agent, implemented as a Rust binary. If you'd like to run your own relayer, we've open sourced the [binary here](https://github.com/hyperlane-xyz/hyperlane-monorepo/tree/main/rust/agents/relayer).
```

Comment:
```suggestion
While permissionless to run, it's up to the message sender to pick which relayer to pay for delivery. [Mailbox](../mailbox.mdx) integrations usually default to always paying the same relayer, but message senders can opt out of this. For convenience, Hyperlane will run an open source and configurable relayer agent, implemented as a Rust binary. If you'd like to run your own relayer, we've open sourced the [binary here](https://github.com/hyperlane-xyz/hyperlane-monorepo/tree/main/rust/agents/relayer).
```

nit: I don't feel strongly, but idk if we wanna explicitly discourage relayer operators